pkg/network/master/egressnp.go

```diff
@@ -0,0 +1,47 @@
+package master
+
+import (
+	"k8s.io/apimachinery/pkg/watch"
+
+	osdnv1 "github.com/openshift/api/network/v1"
+	osdninformers "github.com/openshift/client-go/network/informers/externalversions/network/v1"
+	"github.com/openshift/sdn/pkg/network/common"
+	"github.com/openshift/sdn/pkg/network/master/metrics"
+)
+
+type egressNetworkPolicyManager struct {
+	egressRuleCount int
+}
+
+func newEgressNetworkPolicyManager() *egressNetworkPolicyManager {
+	return &egressNetworkPolicyManager{}
+}
+
+func (enp *egressNetworkPolicyManager) start(informer osdninformers.EgressNetworkPolicyInformer) {
+	informer.Informer().AddEventHandler(
+		common.InformerFuncs(&osdnv1.EgressNetworkPolicy{}, enp.handleAddUpdate, enp.handleDelete))
+}
+
+func (enp *egressNetworkPolicyManager) handleAddUpdate(current, old interface{}, _ watch.EventType) {
+	var change int
+	currentEgressNetworkPolicy, _ := current.(*osdnv1.EgressNetworkPolicy)
+	if old != nil {
+		oldEgressNetworkPolicy, _ := old.(*osdnv1.EgressNetworkPolicy)
+		change = len(currentEgressNetworkPolicy.Spec.Egress) - len(oldEgressNetworkPolicy.Spec.Egress)
+	} else {
+		change = len(currentEgressNetworkPolicy.Spec.Egress)
+	}
+	enp.egressRuleCount += change
```

Comment:
You know what would go great with that code? A unit test! :slightly_smiling_face: